SU2_CFD/src/iteration/CIteration.cpp

```diff
@@ -63,6 +63,17 @@ void CIteration::SetGrid_Movement(CGeometry** geometry, CSurfaceMovement* surfac
       /*--- Already initialized in the static mesh movement routine at driver level. ---*/
```

Comment:
Stale comment, mention why they need to be recomputed

Reply:
Comment is now more clear in 75777c7